packages/analyticsdx-template-lint/src/schemas/variables-schema.json

```diff
@@ -114,6 +114,15 @@
         "max": {
           "doNotSuggest": true
         },
+        "format": {
+          "type": "string",
```

Comment:
This type information should really be down below on the NumberType definition (with `"doNotSuggest": false` down there, otherwise they'll never show up in code completions in the editor) since they're only valid for numbers, and discriminated types in json schema are weird.

Reply:
Good call out, I completely missed everything below this part of the file. I made the change and I think it works correctly now (you get the code completions but only for NumberType variable types)